lib/engine/game/g_1868_wy/game.rb

```diff
@@ -2216,6 +2216,14 @@ def event_close_privates!
         def event_setup_company_price_up_to_face!
           setup_company_price_up_to_face
         end
+
+        def check_connected(route, corporation)
+          route.hexes.any? do |hex|
```

Comment:
each!’?

Reply:
lol I thought I was going to return a bool when I started writing that and I forgot to go back once I saw the error was needed instead